tickets/db.py

```
@@ -1,11 +1,89 @@
-from typing import List
-from tickets.models import TicketEntry
+from pymongo.collection import Collection
+from pymongo.results import InsertOneResult, UpdateResult, DeleteResult
```

Ruff (0.11.9)

`pymongo.results.UpdateResult` imported but unused

Remove unused import

(F401)